examples/ios/Examples/CompleteQuickStart.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  QuickStart.swift
+//  RealmExamples
+//
+//  Created by Chris Bush on 2020-09-15.
+//  Copyright © 2020 MongoDB, Inc. All rights reserved.
+//
+
+import XCTest
+import RealmSwift
+
+private var g_expectation: XCTestExpectation?
+
+import UIKit
+import RealmSwift
+
+class QsTask: Object {
+    @objc dynamic var _id: ObjectId = ObjectId.generate()
+    // When configuring Sync, we selected `_partition` as the partition key.
+    // A partition key is only required if you are using Sync.
+    @objc dynamic var _partition: String = ""
+    @objc dynamic var name: String = ""
+    @objc dynamic var owner: String? = nil
+    @objc dynamic var status: String = ""
+    override static func primaryKey() -> String? {
+        return "_id"
+    }
+
+    convenience init(partition: String, name: String) {
+        self.init()
+        self._partition = partition;
+        self.name = name;
+    }
+}
+
+
+class CompleteQuickStartTest: XCTestCase {
+
+    override func setUp() {
+        let expectation = XCTestExpectation(description: "Log in successfully")
+        app.login(credentials: Credentials.anonymous()) { (user, error) in
+            // Remember to dispatch back to the main thread in completion handlers
+            // if you want to do anything on the UI.
+            DispatchQueue.main.async {
+                guard error == nil else {
+                    print("Login failed: \(error!)")
+                    return
+                }
+
+                print("Login succeeded!");
+
+                expectation.fulfill()
+            }
+        }
+        wait(for: [expectation], timeout: 25.0)
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testCompleteQuickStart() {
+        
+    }
+    
+    func testRunExample() {
+        let expectation = XCTestExpectation(description: "Run complete quick start")
+        // Now logged in, do something with user
+        let user = app.currentUser()
+        let partitionValue = "myPartition"
+
+        var configuration = user!.configuration(partitionValue: partitionValue)
+        configuration.objectTypes = [QsTask.self]
+        Realm.asyncOpen(configuration: user!.configuration(partitionValue: partitionValue)) { (realm, error) in
+            guard let realm = realm else {
+                fatalError(error!.localizedDescription)
+            }
+            // All dogs in the realm
```

Comment:
```suggestion
            // All tasks in the realm
```
...though I must say, I love the vibe of "All dogs in the realm" as a generic phrase for an unfiltered results collection.